packages/mattermost-redux/src/actions/users.ts

```diff
@@ -76,14 +77,22 @@ export function login(loginId: string, password: string, mfaToken = '', ldapOnly
     return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
         dispatch({type: UserTypes.LOGIN_REQUEST, data: null});
 
-        const deviceId = getState().entities.general.deviceToken;
+        const state = getState();
+        const deviceId = state.entities.general.deviceToken;
 
         try {
             await Client4.login(loginId, password, mfaToken, deviceId, ldapOnly);
 
-            const dataFromLoadMe = await dispatch(loadMe());
+            let isSuccessfullyLoggedIn = false;
+            if (isGraphqlEnabled(state)) {
```

Comment:
Do we have at this point enough information in the config to check this? My point being, the config before login is sanitized. Not sure if this piece of information is sanitized or not.

Reply:
Also, should config be reloaded after login (for the same reason) or that is controlled elsewhere?

Reply:
Thanks! That solves my first issue. The second comment is also handled somewhere else?